<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmDeleteDialog</class>
 <widget class="QDialog" name="ConfirmDeleteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Confirm Delete</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>I am sure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ConfirmLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Are you sure you want to delete {table_name}? 
This action cannot be undone.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
